SpikeSimulationTool.m

```diff
@@ -72,27 +73,32 @@ function SpikeSimulationTool(varargin)
 evnts.amplitude_dist_value = 0.5 + (1.5 - 0.5) * rand;      % Value of the new amplitude multiplier
 evnts.amplitude_dist_prob  = 0.2; % Probability of having a change in the amplitude
 
-evnts.prob_start  = floor(0 + ((Naxons/2 - 0) * rand)); % (Recruited) Number of axons that don't start at the beginning. They will randomly start somewhere along the recording.
-evnts.prob_end    = floor(0 + ((Naxons/2 - 0) * rand)); % (Dismissed) Number of axons that don't last the whole recording. They will randomly end somewhere along the recording.
+evnts.prob_start           = floor(0 + ((Naxons/2 - 0) * rand)); % (Recruited) Number of axons that don't start at the beginning. They will randomly start somewhere along the recording.
+evnts.prob_end             = floor(0 + ((Naxons/2 - 0) * rand)); % (Dismissed) Number of axons that don't last the whole recording. They will randomly end somewhere along the recording.
+
 
 %% Run
 % Load the templates matrix
-load(['.' filesep 'templates' filesep 'templates4']);
+d = load(['.' filesep 'templates' filesep 'templates4']);
```

Comment:
Should not have the variable d here since the load function assigns it to a struct